src/components/AdjustVaultForm.tsx

```diff
@@ -0,0 +1,71 @@
+import ErrorWarning from 'svg/error-warning';
+import AmountInput from './AmountInput';
+import StyledDropdown from './StyledDropdown';
+
+const AdjustVaultForm = () => {
+  return (
+    <div className="bg-white font-serif p-8 shadow-[0_40px_40px_-14px_rgba(116,116,116,0.25)] rounded-[20px] w-full">
+      <div className="font-bold mb-4">Adjust Collateral</div>
+      <div className="mb-12 grid grid-cols-2 gap-10">
+        <div className="text-[#9193A5] text-sm col-span-2 lg:col-span-1">
+          Deposit additional collateral or withdraw your existing collateral.
+          Select No Action to leave collateral unchanged.
```

Comment:
"No Action" should be distinguished from the instructional text. Whatever style is used elsewhere for referring to UI options, or at least bold or italicize.

EDIT: I see it's put in quotes on line 38. That works too.

Reply:
Done. In my defense I copy pasted both from figma, not sure why one didn't have quotes 😄